Jenkinsfile

```diff
@@ -33,14 +33,21 @@ def cmake_build(Map conf=[:]){
     def debug_flags = "-g -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=undefined -Wno-option-ignored " + conf.get("extradebugflags", "")
     def build_envs = "CTEST_PARALLEL_LEVEL=4 " + conf.get("build_env","")
     def prefixpath = conf.get("prefixpath","/opt/rocm")
+    def build_type_debug = (conf.get("build_type",'release') == 'debug')
+    def code_conv_enabled = conf.get("codecov", false)
+
     def mlir_args = " -DMIOPEN_USE_MLIR=" + conf.get("mlir_build", "ON")
+    // WORKAROUND_ISSUE_3192 Disabling MLIR for debug builds since MLIR generates sanitizer errors.
+    if (build_type_debug || code_conv_enabled)
+    {
+        mlir_args = " -DMIOPEN_USE_MLIR=OFF"
+    }
```

Comment:
[Notice] @BrianHarrisonAMD This overrides value of "mlir_build", even if it is provided explicitly.

Recommended way:
```groovy
def mlir_build_default = "ON"
if (build_type_debug || code_conv_enabled) {
    mlir_build_default = "OFF"
}
def mlir_args = " -DMIOPEN_USE_MLIR=" + conf.get("mlir_build", mlir_build_default)
```
However, current code is acceptable as a workaround.

/cc @junliume